packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -147,9 +190,68 @@ export const prepareLocalchainAccountKit = (baggage, makeRecorderKit, zcf) => {
         makeTransferAccountInvitation() {
           throw Error('not yet implemented');
         },
+        /** @type {LocalChainAccount['deposit']} */
+        async deposit(payment) {
+          return E(this.facets.helper.owned()).deposit(payment);
+        },
+        /** @type {LocalChainAccount['withdraw']} */
+        async withdraw(amount) {
+          return E(this.facets.helper.owned()).withdraw(amount);
+        },
+        /**
+         * @returns {ChainAddress['address']}
+         */
+        getAddress() {
+          return NonNullish(this.state.address, 'Chain address not available.');
+        },
+        /**
+         * @param {AmountArg} amount
+         * @param {ChainAddress} destination
+         * @param {IBCMsgTransferOptions} [opts] if either timeoutHeight or timeoutTimestamp are not supplied, a default timeoutTimestamp will be set for 5 minutes in the future
+         * @returns {Promise<void>}
+         */
+        async transfer(amount, destination, opts) {
+          trace('Transferring funds from LCA over IBC');
+          // TODO #9211 lookup denom from brand
+          !matches(amount, AmountShape) || Fail`ERTP Amounts not yet supported`;
+
+          // TODO #8879 chainInfo and #9063 well-known chains
+          const { transferChannel } =
+            this.state.agoricChainInfo.connections.get(destination.chainId);
```

Comment:
The `destination: ChainAddress` argument seems to assume we should be able to get `channelId`s for a `chainId`